plugins/glustershd/transactions.go

```diff
@@ -0,0 +1,34 @@
+package glustershd
+
+import (
+	"github.com/gluster/glusterd2/glusterd2/daemon"
+	"github.com/gluster/glusterd2/glusterd2/transaction"
+	"github.com/gluster/glusterd2/glusterd2/volume"
+)
+
+func selfhealdAction(c transaction.TxnCtx, action string) error {
+	glustershDaemon, err := newGlustershd()
+	if err != nil {
+		return err
+	}
+
+	switch action {
+	case "actionStart":
+		err = daemon.Start(glustershDaemon, true)
```

Comment:
If already started, you may have to notify the running glustershd to get the latest volfile. (`sunrpc.FetchSpecNotify(c)`)